helmfile/upstream/fluentd-elasticsearch/values.yaml

```diff
@@ -1,9 +1,9 @@
 image:
-  repository: quay.io/fluentd_elasticsearch/fluentd
+  repository: elastisys/fluentd
 ## Specify an imagePullPolicy (Required)
 ## It's recommended to change this to 'Always' if the image tag is 'latest'
 ## ref: http://kubernetes.io/docs/user-guide/images/#updating-images
-  tag: v3.1.0
+  tag: 3.4.5
```

Comment:
Please don't update this file. This should be **exactly** the same as upstream, and then you should update these values in the values files instead. That way we can super easily track what changes we've made from the upstream chart.

Reply:
Indeed! Will do that next!